pinakes/main/migrations/0038_seed_email_notification.py

```diff
@@ -0,0 +1,132 @@
+from django.db import migrations, transaction
+
+SCHEMA = {
+    "schemaType": "default",
+    "schema": {
+        "fields": [
+            {
+                "label": "Host",
+                "name": "host",
+                "helper_text": "The SMTP host to use for sending email",
+                "isRequired": True,
+                "component": "text-field",
+                "type": "string",
+                "validate": [
+                    {"type": "required-validator"},
+                ],
+            },
+            {
+                "label": "Port",
+                "name": "port",
+                "initialValue": 25,
+                "helper_text": "Port to use for the SMPT server",
+                "component": "text-field",
+                "type": "string",
+                "dataType": "integer",
+                "validate": [
+                    {"type": "required-validator"},
+                    {"type": "min-number-value", "value": 1},
+                ],
+            },
+            {
+                "label": "Username",
+                "name": "username",
+                "helper_text": "Username to authenticate with the SMTP server",
+                "isRequired": False,
+                "component": "text-field",
+                "type": "string",
+            },
+            {
+                "label": "Password",
+                "name": "password",
+                "helper_text": "Password to authenticate with the SMTP server",
+                "isRequired": False,
+                "component": "text-field",
+                "type": "password",
+            },
+            {
+                "label": "Security",
+                "name": "security",
+                "helper_text": "Choose a security connection method to SMTP",
+                "component": "select-field",
+                "isRequired": False,
+                "options": [
+                    {"label": "SSL", "value": "use_ssl"},
+                    {"label": "TLS", "value": "use_tls"},
+                ],
+            },
+            {
+                "label": "SSL Keyfile",
+                "name": "ssl_keyfile",
+                "helper_text": (
+                    "Path to a PEM-formatted private key file to use for the "
+                    "SSL connection"
+                ),
+                "isRequired": False,
+                "component": "text-field",
+                "type": "string",
+            },
+            {
+                "label": "SSL Certfile",
```

Comment:
@bzwei Do we have to allow for self signed certs, verify_ssl=True|False|Ca_File approach

Reply:
no such choice from backend